web/modules/custom/dpl_breadcrumb/dpl_breadcrumb.module

```diff
@@ -37,7 +37,7 @@ function dpl_breadcrumb_preprocess_paragraph__breadcrumb_children(array &$variab
   $paragraph = $variables['paragraph'] ?? NULL;
 
   // In the preview (AKA backend view), we don't want to load dynamics.
-  if (!($paragraph instanceof Paragraph) || $variables['view_mode'] === 'preview') {
+  if (!($paragraph instanceof Paragraph) || $variables['view_mode'] === 'preview' || !$paragraph->hasField('field_show_subtitles')) {
```

Comment:
i think it's a bit extreme to add the check here.

wouldnt it be better to wrap the place you use it (line 54) with an IF instead?

Reply:
Fine by me, altho we do it like this mostly everywhere else i think. 

I moved it to 54: 

```
  if ($paragraph->hasField('field_show_subtitles')) {
    $variables['show_subtitles'] = (bool) $paragraph->get('field_show_subtitles')>value;
  }
```

I think it is a bit strange tho, since the field will be shown for the admin even if this fails 🤷